packages/foreman/rubygem-rsec/rubygem-rsec.spec

```
@@ -1,52 +1,68 @@
-# Generated from rsec-0.4.2.gem by gem2rpm -*- rpm-spec -*-
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
 %global gem_name rsec
 
-Name: rubygem-%{gem_name}
+Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 0.4.3
-Release: 1%{?dist}
+Release: 2%{?dist}
 Summary: Extreme Fast Parser Combinator for Ruby
 Group: Development/Languages
-License: Ruby or BSD 
+License: Ruby or BSD
 URL: http://rsec.herokuapp.com
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
-Requires: ruby(release)
-Requires: ruby(rubygems) 
-BuildRequires: ruby(release)
-BuildRequires: rubygems-devel 
+
+# start specfile generated dependencies
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby >= 1.9.1
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby >= 1.9.1
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
 BuildArch: noarch
-Provides: rubygem(%{gem_name}) = %{version}
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+# end specfile generated dependencies
 
 %description
 Easy and extreme fast dynamic PEG parser combinator.
 
 
 %package doc
-Summary: Documentation for %{name}
+Summary: Documentation for %{pkg_name}
 Group: Documentation
-Requires: %{name} = %{version}-%{release}
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
 BuildArch: noarch
 
 %description doc
-Documentation for %{name}.
+Documentation for %{pkg_name}.
 
 %prep
+%{?scl:scl enable %{scl} - << \EOF}
 gem unpack %{SOURCE0}
+%{?scl:EOF}
 
 %setup -q -D -T -n  %{gem_name}-%{version}
 
+%{?scl:scl enable %{scl} - << \EOF}
 gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+%{?scl:EOF}
 
 %build
 # Create the gem as gem install only works on a gem file
+%{?scl:scl enable %{scl} - << \EOF}
 gem build %{gem_name}.gemspec
+%{?scl:EOF}
 
 # %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
 # by default, so that we can move it into the buildroot in %%install
+%{?scl:scl enable %{scl} - << \EOF}
 %gem_install
+%{?scl:EOF}
 
 %install
 mkdir -p %{buildroot}%{gem_dir}
-cp -a .%{gem_dir}/* \
+cp -pa .%{gem_dir}/* \
```

We simplified the template: https://github.com/theforeman/foreman-packaging/commit/f02a98ade2c230f3968cd12e0ac220262f10109c

```suggestion
cp -a .%{gem_dir}/* \
```